### ISSUE TITLE
optimize the result header of the query 

### ISSUE THREAD

**Issue by jacketzc** [mtxr/vscode-sqltools#610]
I hope to optimize the result header of the query because it is too long

**Comment by mtxr** [comment]
Any suggestions?

**Comment by iam-mhaseeb** [comment]
Every common db IDE uses horizontal scroll bar.

**Comment by mtxr** [comment]
I'm still not picturing it. Can you share some images? I use DBeaver/DataGrip as inspiration, and I don't get it

**Comment by iam-mhaseeb** [comment]
Exactly, the horizontal scrollbar does the trick. DBeaver & DataGrip also uses a horizontal scrollbar.
Take this image, extra columns are hidden and to view it we have to move horizontal scrollbar to the right which is actually not clearly visible but it will give you an idea.
https://d3nmt5vlzunoa1.cloudfront.net/datagrip/files/2015/12/tableeditor.gif
